clients/web/src/views/widgets/MetadataWidget.js

```diff
@@ -13,7 +13,10 @@ girder.views.MetadataWidget = girder.View.extend({
 
     initialize: function (settings) {
         this.item = settings.item;
+        this.fieldName = settings.fieldName || 'meta';
         this.accessLevel = settings.accessLevel;
+        this.onMetadataEdited = settings.onMetadataEdited;
```

Comment:
Couldn't this just be ``this.onMetadataEdited = settings.onMetadataEdited || this.item.addMetadata;``? This removes the need to do a conditional call below (where both functions are passed the exact same arguments).